__tests__/data/mautic.json

```diff
@@ -0,0 +1,842 @@
+[
```

Comment:
Test cases for these red marked parts are absent. Please add them
<img width="1212" alt="Screenshot 2022-08-21 at 3 15 14 PM" src="https://user-images.githubusercontent.com/60211312/185785223-f71b3bb2-62ef-4635-a2e9-a122968ad4ef.png">